README.md

```diff
@@ -336,6 +335,23 @@ allow operation with the unoptimized Javascript files.
 Debugging can be enabled by creating the file
 `specify7/specifyweb/settings/debug.py` with the contents, `DEBUG = True`.
 
+**Backend:**
+Enabling `SP7_DEBUG` enables DEBUG mode for Django
+<https://docs.djangoproject.com/en/3.2/ref/settings/#debug>
+
+The Specify backend also special behavior while in DEBUG mode:
+- When the backend encounters an error, Django returns an error message containing comprehensive information of the error and potentially sensitive system information
+- Specify validates the WorkBench upload plan when fetching the WorkBench's upload results
+   - From the `/api/workbench/upload_results/(?P&lt;ds_id&gt;\d+)/` endpoint
+- Specify adds an indent to the datamodel returned from the `context/datamodel.json` endpoint
+- (DISABLED BY DEFAULT) Specify allows any endpoint/view to be profiled by adding a `/?prof` query parameter
+
+**Frontend:**
```

Comment:
Note, enabling DEBUG mode using SP7_DEBUG does not affect front-end at all!

Front-end debug mode is enabled separately:

https://github.com/specify/specify7/blob/bd5f5bc51fc03907936bcbc15e544afe72a36cc7/docker-compose.yml#L72

https://github.com/specify/specify7/blob/bd5f5bc51fc03907936bcbc15e544afe72a36cc7/Dockerfile#L37

(`-w` means watch for changes and rebuild front-end on changes)

See documentation at https://webpack.js.org/api/cli/

Reply:
And unlike with python where debug mode can be enabled/disabled at will, front-end files are compiled from typescript into javascript and production/development mode must be specified during compilation.

In production, webpack is not running at all (it's only present at all when the docker image is being built by github action, and after that the build artifacts are copied into docker image, but not webpack itself as it's not needed anymore)